hosting_docs/source/installation/migration/1-migrating-project.rst

```diff
@@ -25,41 +25,50 @@ switchover to the new environment.
 1. Switch mobile devices to a proxy URL
 ---------------------------------------
 
-Each application maintains a series of URLs used for the various requests made
-by the mobile devices. Since rolling out an app update to all mobile devices
-takes time, it's best to not rely on that when switching server environments.
-Instead, we direct apps to a custom URL which directs requests to the original
-server, then after the migration, to the new server. This switch happens at the
-DNS level, without requiring a change on each device.
+Each application maintains a series of URLs pointing to CommCareHQ environment used for
+various requests made by the mobile devices. Migrating to a new web address requires updating these
+URLs in all the mobilie devices at the time of switching the environments after the data is migrated. Since rolling out an app update to every mobile device of the project
+takes time during which the site needs to be offline, it will result in a long downtime for the project. Hence, if your project have any more than a couple of devices, it's best not to do it this way.
```

Comment:
```suggestion
takes time during which the site needs to be offline, it will result in a long downtime for the project. Hence, if your project has any more than a couple of devices, it's best not to do it this way.
```